docs/content/3.components/dropdown-menu.md

```diff
@@ -32,6 +32,7 @@ Use the `items` prop as an array of objects with the following properties:
 - [`slot?: string`{lang="ts-type"}](#with-custom-slot)
 - `onSelect?(e: Event): void`{lang="ts-type"}
 - [`onUpdateChecked?(checked: boolean): void`{lang="ts-type"}](#with-checkbox-items)
+- `children?: DropdownMenuItem[]`{lang="ts-type"}
```

Comment:
Actually... I just realized, this should probably be `DropdownMenuItem[][]`, right? :thinking:

Reply:
It should be `DropdownMenuItem[] | DropdownMenuItem[][]`. 

Would you mind also adding this in `context-menu.md` and `navigation-menu.md` for consistency? 😬